### DISCUSSION TITLE
Changing "Visual" indentation

### DISCUSSION THREAD

**Discussion by Penaz91** [nvim-neorg/neorg#1054] [Q&A]
Hello!

I've been eyeing this project for a while and today I decided to jump and use it as my daily driver for note-taking and organization.

I've had a bit of a hard time configuring it, but in the end I managed to make things work. There is one thing that is giving me a hard time processing neorg files: indentation.

For me 1 space is not enough to be able to distinguish between indentation levels in a nested todo list (I don't see very well, and I use A LOT of nested lists). Would it be possible to configure Neorg (or its concealer) to change the indent amount (visually speaking) for all items (headers, lists, nested lists, etc...)?

I tried looking through the wiki and the discussions, but it doesn't seem there is an option specific for this.

Thank you in advance for your help!

**Comment by max397574** [comment]
now that we got anticonceal this would theoretically be possible but currently it isn't from the neorg side

**Reply by champignoom** [reply]
https://github.com/nvim-neorg/neorg/pull/1055 but soon

**Comment by pysan3** [comment]
Just a random idea but making something like indentLine to add virtual lines might be a solution?

https://github.com/Yggdroot/indentLine

**Comment by RomaLzhih** [comment]
Strongly agree, 1 space indention is too small to distinguish between other items, especially the lists.
